hooks/post_gen_project.py

```diff
@@ -18,7 +24,7 @@
 )
 PSQL_COMMAND = '{{cookiecutter.psql_command}}'
 VIRTUALENV = '{{cookiecutter.virtualenv}}'
-VIRTUALENV_COMMAND = '{{cookiecutter.virtualenv_command}}'
+VIRTUALENV_COMMAND = '{{cookiecutter.virtualenv_command}}'.replace('%PYTHON%', python_path())
```

Comment:
Clever. Maybe add `/opt/local/bin/python3` for MacPorts users. By the way, what happens if I override the string so it doesn't contain `%PYTHON%` anymore?